pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
     "matplotlib>=3.7.1",
     "scipy>=1.10.1",
     "pandas>=1.5.3",
-    "numpy>=1.24.4",
+    "numpy>=1.25.0",
```

Comment:
Please change version in `pyaerocom_env.yml` as well.

Reply:
Thanks, was not aware of that file